### ISSUE TITLE
Add easier option for toggling SOCKS5 proxy

### ISSUE THREAD

**Issue by asakura42** [TrackerControl/tracker-control-android#319]
I use SOCKS5 proxy sometimes but don't want to use it all the time. The current method is quite difficult. You have to go into the settings, press the button at the bottom of the page. Why not make the switch easier?

**Comment by kasnder** [comment]
Yeah, I agree. Thanks for raising this! For now, you could use two versions of TrackerControl, the one from F-Droid and the one from GitHub, and toggle between the two by enabling one of them (and thereby disabling the other)?

**Comment by siebenbrodt** [comment]
I join the request. It would be great to have a widget for the SOCKS5 switch.
